Example Apps/Example ObjC/RemoteControlManager.h

```diff
@@ -0,0 +1,29 @@
+//
+//  RemoteControlManager.h
+//  SmartDeviceLink-Example-ObjC
+//
+//  Created by Beharry, Justin (J.S.) on 8/1/22.
+//  Copyright © 2022 smartdevicelink. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class SDLManager;
+@class SDLSoftButtonObject;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface RemoteControlManager : NSObject
+
+@property (assign, nonatomic, getter=isPermissionEnabled, readonly) BOOL permissionEnabled;
```

Comment:
```suggestion
@property (assign, nonatomic, readonly, getter=isEnabled) BOOL enabled;
```

The style for the project is (1) memory storage, (2) atomicity, (3) readability, (4) custom getter

I don't remember if there are other options.